lib/accessibility.js

```diff
@@ -0,0 +1,51 @@
+const Eval = require('./eval')
+const __devtron = window.devtron
```

Comment:
Hmm, I'm not sure this should be defined here, the function passed to `Eval.execute` runs inside the renderer process so it will not have access to this variable here.